tunnel-server/index.ts

```diff
@@ -55,10 +50,10 @@ const sshServer = createSshServer({
   log: sshLogger,
   sshPrivateKey,
   socketDir: '/tmp', // TODO
-  onPipeCreated: async (clientId, remotePath, localSocket) => {
+  onPipeCreated: async ({clientId, remotePath, localSocketPath, publicKey, access}) => {
```

Comment:
I think that for consistency, both onPipeCreated and onPipeDestroyed should use props style signature